packages/client/hmi-client/src/components/widgets/tera-modal.vue

```diff
@@ -112,7 +112,7 @@ main > section {
 }
 
 .content {
-	max-height: 80vh;
+	max-height: 70vh;
```

Comment:
Were you the one who made this 80vh previously? Wondering if the reduction will impact someone else.

Reply:
Yes I made it from 65 -> 80.  This was causing some visual issues when the component got too big (pushing the footer button down past the modal space).  70 seems to be a sweet spot